easybuild/easyconfigs/g/Gurobi/Gurobi-9.0.0-intel-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Gurobi'
+version = '9.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.gurobi.com'
+description = """The Gurobi Optimizer is a state-of-the-art solver for mathematical programming.
+The solvers in the Gurobi Optimizer were designed from the ground up to exploit modern 
+architectures and multi-core processors, using the most advanced implementations of the 
+latest algorithms."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+# registration is required
```

Comment:
Comment is slightly misleading. You don't need to register to download (worked for me with --from-pr)

Reply:
Oops, that comment has survived from version 7.X where that was the case.  Removed.